opencti-platform/opencti-front/src/utils/platformModulesHelper.ts

```diff
@@ -41,8 +41,9 @@ const isFeatureEnable = (
   id: string,
 ) => {
   const flags = settings.platform_feature_flags ?? [];
-  const feature = flags.find((f) => f.id === id);
-  return feature === undefined || feature.enable === true;
+  // config can target all FF available with special FF id "*"
+  const feature = flags.find((f) => f.id === id || f.id === '*');
```

Comment:
Hm, I would have done a big (if flags.includes('*')){ return true; } to be more explicite

Reply:
yep